nucypher/network/nodes.py

```diff
@@ -54,16 +55,16 @@
 from nucypher.blockchain.eth.registry import BaseContractRegistry
 from nucypher.config.constants import SeednodeMetadata
 from nucypher.config.storages import ForgetfulNodeStorage
-from nucypher.crypto.api import InvalidNodeCertificate, recover_address_eip_191, verify_eip_191
 from nucypher.crypto.kits import UmbralMessageKit
 from nucypher.crypto.powers import DecryptingPower, NoSigningPower, SigningPower
 from nucypher.crypto.splitters import signature_splitter
-from nucypher.crypto.umbral_adapter import Signature
+from nucypher.crypto.utils import recover_address_eip_191, verify_eip_191
 from nucypher.network import LEARNING_LOOP_VERSION
 from nucypher.network.exceptions import NodeSeemsToBeDown
 from nucypher.network.middleware import RestMiddleware
 from nucypher.network.protocols import SuspiciousActivity
 from nucypher.utilities.logging import Logger
+from umbral.signing import Signature
```

Comment:
Rebase artefact? Should be imported from `umbral_adapter`.

Reply:
Thanks.  I think I rebased this PR before realizing that I needed to import from umbral adapter.  I'll make an update.

Reply:
✔️